### PR TITLE
Add new attributes into Template Weather

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

This PR adds the newly added attributes into Template Weather
- `wind_gust_speed`
- `cloud_coverage`
- `dew_point`
- `apparent_temperature`

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/95100
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/weather.template.markdown [hunk 1]
```diff
@@ -56,6 +56,14 @@ temperature_template:
...
+dew_point_template:
+  description: The current dew point.
+  required: false
+  type: template
+apparent_temperature_template:
+  description: The current apparent (feels-like) temperature.
+  required: false
+  type: template
 temperature_unit:
   description: Unit for temperature_template output. Valid options are °C, °F and K.
```
Comment (c0ffeeca7) [inline source/_integrations/weather.template.markdown]:
```suggestion
  description: Unit for temperature_template output. Valid options are °C, °F, and K.
```
We use the Oxford comma.

File: source/_integrations/weather.template.markdown [hunk 2]
```diff
@@ -80,6 +88,10 @@ wind_speed_template:
   description: The current wind speed.
   required: false
   type: template
+wind_gust_speed_template:
+  description: The current wind gust speed.
+  required: false
+  type: template
 wind_speed_unit:
   description: Unit for wind_speed_template output. Valid options are m/s, km/h, mph, mm/d, in/d, in/h.
```
Comment (c0ffeeca7) [inline source/_integrations/weather.template.markdown]:
```suggestion
  description: Unit for wind_speed_template output. Valid options are m/s, km/h, mph, mm/d, in/d, and in/h.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

**Review by c0ffeeca7 — APPROVED** [review]
LGTM. Thank you, @gjohansson-ST :+1:

**Review by frenck — APPROVED** [review]
Thanks, @gjohansson-ST 👍 

../Frenck

**Pull request merged**